test/models/AppDataFactory.ts

```diff
@@ -3,12 +3,12 @@ import { Attachment } from 'app/models/Attachment';
 import { PenaltyIdentifier } from 'app/models/PenaltyIdentifier';
 import { Reasons } from 'app/models/Reasons';
 
```

Comment:
The reason for these changes was because these variables can be mutated by the tests making outcome unpredictable.
Example:

```ts
const appealA = APPEAL_WITH_ATTACHMENTS;
const appealB = APPEAL_WITH_ATTACHMENTS;

delete appealA.penaltyIdentifier
console.log(appealB.penaltyIdentifier) // undefined
```